packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.test.js

```diff
@@ -16,4 +18,38 @@ describe('<SliderUnstyled />', () => {
     refInstanceof: window.HTMLSpanElement,
     testComponentPropWith: 'span',
   }));
+
+  it('forwards style props as DOM attributes', () => {
+    let styleProps = null;
+    let theme = null;
+
+    const Root = React.forwardRef(
+      ({ styleProps: stylePropsProp, theme: themeProp, ...rest }, ref) => {
+        styleProps = stylePropsProp;
+        theme = themeProp;
+        return <span ref={ref} {...rest}></span>;
+      },
+    );
+
+    render(<SliderUnstyled components={{ Root }} />);
+
+    expect(styleProps).not.to.equal(null);
+    expect(styleProps).not.to.equal(null);
```

Comment:
One of these is probably meant to be `theme`?

Reply:
Fixed already :)